pull_request_template.md

```diff
@@ -6,9 +6,8 @@
 <!--- If it fixes an open issue, please link to the issue here, using the 'fixes #<issue>' syntax. -->
 
 ## How Has This Been Tested?
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
+- [ ] New automated tests added
```

Comment:
Better, but doesn't the coverage testing take care of this? I'm not sure what the added value of this bit is

Reply:
Sure we could take it out it's more when adding things you can't test automatically like PR templates